cfy_manager/main.py

```diff
@@ -753,6 +756,9 @@ def install(verbose=False,
             # install then we shouldn't configure
             if not component.skip_installation:
                 component.configure()
+        for component in components:
```

Comment:
don't we want that to stay separate?

Reply:
good point, no reason for this to be in this PR